lib/Doctrine/Migrations/Configuration/Loader/PHPFileLoader.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations\Configuration\Loader;
+
+use Doctrine\Migrations\Configuration\Configuration;
+use Doctrine\Migrations\Configuration\Exception\FileNotFound;
+use function assert;
+use function file_exists;
+use function is_array;
+
+/**
+ * @internal
+ */
+final class PHPFileLoader extends AbstractFileLoader
+{
+    /** @var ArrayLoader */
+    private $arrayLoader;
+
+    public function __construct(?ArrayLoader $arrayLoader = null)
+    {
+        $this->arrayLoader = $arrayLoader ?: new ArrayLoader();
+    }
+
+    /**
+     * @param mixed $file
+     */
+    public function load($file) : Configuration
+    {
+        if (! file_exists($file)) {
+            throw FileNotFound::new();
```

Comment:
What about something like this?
```suggestion
            throw FileException::notFound();
```

all the time `new()` feels a bit weird.

Reply:
are you suggesting to drop the `FileNotFound` class in favor of a more generic `FileException`?